apps/datahub/src/app/record/header-record/header-record.component.html

```diff
@@ -1,36 +1,63 @@
-<header class="w-full h-[231px]" [style.background]="backgroundCss">
+<header class="w-full h-[255px]" [style.background]="backgroundCss">
   <div
-    class="h-full container-lg mx-auto flex flex-col justify-center relative"
+    class="h-full container-lg mx-auto flex flex-col justify-between relative"
   >
-    <gn-ui-favorite-star
-      *ngIf="metadata?.uniqueIdentifier"
-      [record]="metadata"
-      class="absolute text-background top-[0.85em] right-[0.85em]"
-      [style.color]="foregroundColor"
-      [style.--star-toggle-enabled-color]="'var(--color-primary)'"
-      [style.--star-toggle-disabled-color]="foregroundColor"
-    ></gn-ui-favorite-star>
+    <div class="flex flex-row justify-between content-center mt-3">
+      <div class="ml-4">
+        <gn-ui-navigation-button
+          (click)="back()"
+          [label]="'datahub.search.back' | translate"
+          [icon]="'arrow_back'"
+        >
+        </gn-ui-navigation-button>
+      </div>
+      <div class="flex flex-row mr-6 content-center gap-3">
+        <gn-ui-favorite-star
+          *ngIf="metadata?.uniqueIdentifier"
+          [record]="metadata"
+          [displayCount]="false"
+          class="flex text-background content-center"
+          [style.color]="foregroundColor"
+          [style.--star-toggle-enabled-color]="'var(--color-primary)'"
+          [style.--star-toggle-disabled-color]="foregroundColor"
+        ></gn-ui-favorite-star>
+        <gn-ui-language-switcher
```

Comment:
The language switcher should be displayed under a condition, isn't it ?
Is it the case or not ?

Reply:
In `home-header`, yes, there is a condition for its display. But in the mockups (nor in the ticket), this wasn't specified. I assumed that the switcher had to be there at all times.